packages/suite/src/components/firmware/ReconnectDevicePrompt.tsx

```diff
@@ -201,30 +143,25 @@ const ReconnectStep = ({ order, active, dataTest, children }: ReconnectStepProps
 
 const RebootDeviceGraphics = ({
     device,
-    method,
-    requestedMode,
+    isManualRebootRequired,
 }: {
-    device?: TrezorDevice;
-    method: RebootMethod;
-    requestedMode: RebootRequestedMode;
+    device?: Device;
+    isManualRebootRequired: boolean;
 }) => {
-    if (method === 'automatic') {
+    if (!isManualRebootRequired) {
         return device ? <StyledConfirmImage device={device} /> : null;
     }
 
     const deviceModelInternal = device?.features?.internal_model;
 
     // T1B1 bootloader before firmware version 1.8.0 can only be invoked by holding both buttons
     const deviceFwVersion = device?.features ? getFirmwareVersion(device) : '';
-    let type: AnimationDeviceType = requestedMode === 'bootloader' ? 'BOOTLOADER' : 'NORMAL';
-    if (
-        type === 'BOOTLOADER' &&
+    const type =
         deviceModelInternal === DeviceModelInternal.T1B1 &&
         semver.valid(deviceFwVersion) &&
         semver.satisfies(deviceFwVersion, '<1.8.0')
-    ) {
-        type = 'BOOTLOADER_TWO_BUTTONS';
-    }
+            ? 'BOOTLOADER_TWO_BUTTONS'
+            : 'BOOTLOADER';
```

Comment:
this could have beeen transefered to the new events in connect as well. just leaving a note and maybe we will do it, maybe not.

Reply:
Sounds good.

Reply:
Maybe at least todo to move it to connect